recipes/dbnomics/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "DBnomics" %}
+{% set version = "1.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/DBnomics-{{ version }}.tar.gz
+  sha256: 02e6039d10860f9d7dc99cd45ffe548934d4e1d373445d1a00b12fb057a0a6ab
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pytest-runner
+    - pip
+  run:
+    - python >=3.6
+    - requests >=2.18.4
+    - pandas >=0.21
+    # check https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#export-runtime-requirements
+
+test:
+  imports:
+    - dbnomics
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://git.nomics.world/dbnomics/dbnomics-python-client
+  summary: DBnomics Python Client
+  license: AGPL-3.0-only
```

Looks like it's actually ["or later"](https://git.nomics.world/dbnomics/dbnomics-python-client/-/blob/master/dbnomics/__init__.py#L10):
```suggestion
  license: AGPL-3.0-or-later
```